src/index.html

```diff
@@ -69,10 +69,36 @@ <h6>Fallidos</h6>
       <button class="btn btn-block btn-primary" onclick="navigate('home')">Go to home</button>
     </section>
 
-    <section class="tutorial screen center-items" id="tutorialScreen">
-      <h2>Tutorial</h2>
+    <section class="tutorial screen  justify-content-between tutorial-background" id="tutorialScreen">
```

Comment:
```suggestion
    <section class="tutorial screen justify-content-between tutorial-background" id="tutorialScreen">
```
Cuidado con los espaciados entre las clases